app/controllers/user/teams_controller.rb

```diff
@@ -8,7 +8,13 @@ def index
 
   def show
     @team = current_user.teams.find_by_id(params[:id].to_i)
-    @players =  @team.players
+    if team_player_params[:benched]
+      team_player = TeamPlayer.find(team_player_params[:team_player])
+      team_player.update_attribute(:benched, team_player_params[:benched])
+      @players =  @team.players
+    else
+      @players =  @team.players
+    end
```

Comment:
Line 14 could be the end of the if statement, and then `@players = @team.players could be after the if block.